src/coordsim/forwarders/link_load_forwarder.py

```diff
@@ -0,0 +1,9 @@
+class LinkLoadForwarder(object):
+    """
+    This is the default forwarder class. It takes the shortest path.
```

Comment:
What's this?

Reply:
This is where I will put the forwarder class that has link loads. It's just a placeholder for now.